glpk/bld.bat

```diff
@@ -0,0 +1,14 @@
+cd w%ARCH%
+copy config_VC config.h
+
+nmake /f Makefile_VC
+
+mkdir %PREFIX%\bin

```

Comment:
Besides @snorfalorpagus published his own glpk recipe five days ago and includes some testing, so perhaps he wants to weigh in. Otherwise I'm just going to edit `bld.bat` as little as possible.